core/src/main/java/org/apache/spark/shuffle/api/ShuffleDriverComponents.java

```diff
@@ -62,12 +62,23 @@ default void registerShuffle(int shuffleId) {}
    */
   default void removeShuffle(int shuffleId, boolean blocking) {}
 
+  /**
+   * This method is for compatibility with older implementations
+   * the implementation should just directly return true
+   * when use distributed filesystem or in a disaggregated shuffle cluster
+   */
```

Comment:
Shall we keep the origin comments here and add these new comments as `Note: ...` at last.

Reply:
ok